### PR TITLE
Added prims-mst

### LINKED ISSUE #284: [Feature Request]  Add Prim’s Minimum Spanning Tree (MST) to greedy
### Is your feature request related to a problem? Please describe.
There is no Prim’s Minimum Spanning Tree (MST) in greedy

### Describe the solution you'd like
add it to greedy 

### Describe alternatives you've considered
na

### Additional context

---

**Would you like to work on this feature?**  
- [x] Yes  
- [ ] No

### PR DESCRIPTION
**Pull request by jvkousthub**
### Description
Added Prim’s Minimum Spanning Tree (MST) to greedy algorithms

Fixes #284

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/greedy-algorithms/Prim’s-Minimum-Spanning-Tree.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+### Example:
+
+Consider the following graph:
+
+---
+id: prims-mst
+title: Prim's Minimum Spanning Tree (MST)
+sidebar_label: Prim's MST
+description: "This repository implements Prim's algorithm for finding the Minimum Spanning Tree (MST) of a graph, using a greedy approach to minimize the total edge weight."
+tags: [dsa, algorithms, greedy algorithms, graph algorithms, minimum spanning tree]
```
Comment (ajay-dhangar) [inline docs/greedy-algorithms/Prim’s-Minimum-Spanning-Tree.md]: Check out your code carefully

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
Your code is incorrect because it does not follow our syntax and formatting requirements. Please ensure that your code follows our guidelines before submitting it.

**Comment by jvkousthub** [comment]
@ajay-dhangar Updated the Prims MST

File: docs/greedy-algorithms/Prim’s-Minimum-Spanning-Tree.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+id: prims-mst
```
Comment (ajay-dhangar) [inline docs/greedy-algorithms/Prim’s-Minimum-Spanning-Tree.md:2]: before line number `1` you need add ```---```

**Review by ajay-dhangar — CHANGES_REQUESTED** [review]
You need to add ```---``` at the beginning of your text for our documentation format.

File: docs/greedy-algorithms/Prim’s-Minimum-Spanning-Tree.md (hunk 2 above)
Comment (jvkousthub) [reply docs/greedy-algorithms/Prim’s-Minimum-Spanning-Tree.md:2]: @ajay-dhangar done

**Comment by jvkousthub** [comment]
@ajay-dhangar done

**Review by ajay-dhangar — APPROVED** [review]
Done

**Pull request merged**